src/rocstar.js

```diff
@@ -0,0 +1,223 @@
+import _ from 'lodash';
+import Keyring from '@polkadot/keyring';
+import BN from 'bn.js';
+import moment from 'moment';
+import TuringHelper from './common/turingHelper';
+import ShibuyaHelper from './common/shibuyaHelper';
+import { sendExtrinsic, getDecimalBN, listenEvents } from './common/utils';
+import { TuringStaging, Rocstar } from './config';
+import Account from './common/account';
+import 
+
+// One XCM operation is 1_000_000_000 weight - almost certainly a conservative estimate.
+// It is defined as a UnitWeightCost variable in runtime.
+const TURING_INSTRUCTION_WEIGHT = 1000000000;
+const MIN_BALANCE_IN_PROXY = 10; // The proxy accounts are to be topped up if its balance fails below this number
+const SHIBUYA_TOKEN_ID_ON_TURING = 4;
+const TASK_FREQUENCY = 3600;
+const LISTEN_EVENT_DELAY = 3 * 60;
+
+const keyring = new Keyring({ type: 'sr25519' });
+
+const scheduleTask = async ({
```

Comment:
@imstar15 Rocstar needs to re-use the code of Shibuya. Please see #26 for my change suggestions.